### PR TITLE
[docs] add go and python sdk links

### PR DESCRIPTION
**Pull request by zihehuang**
## Description 

adding links to go and python sdks

## Test Plan 

visual inspection + manual test

---
If your changes are not user-facing and do not break anything, you can skip the following section. Otherwise, please briefly describe what has changed under the Release Notes section.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: docs/content/sidebars/references.js [hunk 1]
```diff
@@ -69,17 +69,27 @@ const references = [
...
 		},
 		items: [
+			'references/rust-sdk',
 			{
 				type: 'link',
 				label: 'Sui TypeScript SDK',
 				href: 'https://sdk.mystenlabs.com/typescript',
 			},
+      {
+				type: 'link',
```
Comment (ronny-mysten) [inline docs/content/sidebars/references.js:79]: nit: alphabetize perhaps?
dApp
Go
Py
rust
ts

**Review by ronny-mysten — APPROVED** [review]
lgtm

File: docs/content/sidebars/references.js (hunk 1 above)
Comment (ronny-mysten) [reply docs/content/sidebars/references.js:79]: The parent page would need to do this, too

Comment (zihehuang) [reply docs/content/sidebars/references.js:79]: Makes sense

**Pull request merged**